### DISCUSSION TITLE
changing Maxtimeout in docker for jackett call

### DISCUSSION THREAD

**Discussion by shaavik** [FlareSolverr/FlareSolverr#59] [Q&A]
When using curl commande with huge timeout, the result takes more than 60000 ms :
INFO REQ-12 Successful response in 93.188 s

But it seems like the call from Jackett use default parameters. Jackett team told me to post the problem here.

So, I can't get a correct response in time with default parameter (60000).
I want to set it from docker environnement variable. How can I set it ?
I have to use it in docker image, because the windows installation with nodejs fails.

**Comment by ngosang** [comment]
> When using curl commande with huge timeout, the result takes more than 60000 ms :
INFO REQ-12 Successful response in 93.188 s

There was a bug in FlareSolver 1.2.0. It's fixed in v1.2.1 FlareSolverr will respect the 60s timeout from that version.

> I want to set it from docker environnement variable. How can I set it ?

This value is hardcoded so it can't be changed in Jackett, neither Docker image. At this moment I don't see the need to increase this value, but if more people ask for it, I can make it configurable.
